react/features/polls/middleware.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+
+import { MiddlewareRegistry } from '../base/redux';
+import {
+    END_POLL,
+    START_POLL,
+    VOTE_POLL
+} from './actionTypes';
+import {
+    addPoll,
+    showPollStartNotification,
+    showPollEndNotification,
+    updateVote,
+    finishPoll
+} from './actions';
+import { isButtonEnabled } from '../toolbox';
+import { CONFERENCE_JOINED } from '../base/conference';
+import { JitsiConferenceEvents } from '../base/lib-jitsi-meet';
+
+declare var APP: Object;
+declare var interfaceConfig : Object;
+
+MiddlewareRegistry.register(store => next => action => {
+    switch (action.type) {
+    case CONFERENCE_JOINED:
+        typeof APP === 'undefined'
+            || _addPollMsgListener(action.conference, store);
+        break;
+    case START_POLL: {
+        if (APP !== 'undefined') {
```

Comment:
You'll need to make sure to use typeof if you don't want to break mobile. That and/or make sure this middleware only gets imported by web.